### PR TITLE
Added control statements

### PR DESCRIPTION
**Pull request by YamanBoztepe**
i added a control statements

### CODE REVIEW

File: helloWorld.swift [hunk 1]
```diff
@@ -1,5 +1,5 @@
 let exampleText = "a"
 
-if exampleText == "a" {
+if exampleText == "a" && exampleText != "b" {
```
Comment (YamanBoztepeAppcent) [inline helloWorld.swift:3]: Can you add a control state for "c"?

**Review by YamanBoztepeAppcent — COMMENTED** [review]
add a control statement for c

File: helloWorld.swift (hunk 1 above)
Comment (YamanBoztepe) [reply helloWorld.swift:3]: sure

**Comment by YamanBoztepe** [comment]
i will also add control statement for "d"

File: helloWorld.swift (hunk 1 above)
Comment (YamanBoztepeAppcent) [inline helloWorld.swift:3]: add exampleText == "d"

**Pull request closed without merging**